### DISCUSSION TITLE
Possible to alert on a value changing?

### DISCUSSION THREAD

**Discussion by G0nz0uk** [grafana/grafana-csv-datasource#80] [Q&A]
Hello,

I've got many CSVs running in Grafana with some nice coloured thresholds showing.  The next step is alerting, is it possible?

**Comment by marcusolsson** [comment]
Should be! Let me know if you run into issues!

**Comment by Kortenbach** [comment]
@marcusolsson ,
Hi Marcus, I have tried to create an alert with a CSV as data source but it seems impossible. I get an error because the data is not "wide" format. I've tried many different table formats but can't get it working. The data is not evaluated/displayed only error message.

**Reply by Prashantped** [reply]
trying to create alert for a tabular dashboard, not able to figure out how to do it.....
